lib/mhv_logging/configuration.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+require 'rx/configuration'
+module MHVLogging
+  # Configuration class used to setup the environment used by client
+  class Configuration < Rx::Configuration
```

Comment:
Does breakers need to distinguish it? Rx and MHVLogging are technically the same server is my understanding, so if one is down, it would follow that the other is down. Perhaps I should just use the same Rx breaker then maybe?

Only reason why this client is not just using Rx is that it didn't make a whole lot of sense for me to invoke a client for prescriptions from secure messaging, and of course the fact that the sessions, for whatever reason are distinct between SM and Rx+MHVLogging